blocks/layout-grid/src/grid.scss

```diff
@@ -24,7 +24,7 @@
 	}
 
 	.wp-block {
-		max-width: none;
+		max-width: 100%;
```

Comment:
I could swear we added the "none" for a reason. Possibly due to some editor styles assigning various block max-width properties in a sweeping effort in order to accommodate the wide/full-wide configuration.

However if that's the case, then explicitly setting it to 100% is the correct solution, as opposed to removing the rule.

Let's keep an eye on the Twenty themes, but otherwise this seems correct.